### PR TITLE
Jasmine and filterByAge challenge

### PR DESCRIPTION
**Pull request by RominaFior**

### CODE REVIEW

File: staff/RominaFior/js/challenges/box.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (agandia9) [inline staff/RominaFior/js/challenges/box.js:1]: use strict.

File: staff/RominaFior/js/challenges/concat.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/*
```
Comment (agandia9) [inline staff/RominaFior/js/challenges/concat.js:1]: mmmh????

File: staff/RominaFior/js/challenges/scripts/Array.prototype.toRandomCase.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+Array.prototype.toRandomCase = function (){
```
Comment (agandia9) [inline staff/RominaFior/js/challenges/scripts/Array.prototype.toRandomCase.js:1]: use strict.

File: staff/RominaFior/js/challenges/scripts/find.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+'use strict'
+'use strict';
```
Comment (agandia9) [inline staff/RominaFior/js/challenges/scripts/find.js:2]: double use strict. 🤨😝

**Comment by manuelbarzi** [comment]
please, review previous comments and take action.

**Pull request merged**